### DISCUSSION TITLE
请问如何让前后chapter连起来而不用另起一页？

### DISCUSSION THREAD

**Discussion by EF-Lin233** [nju-lug/NJUThesis#80] [Q&A]
例如，当我输入
\input{chapters/introduction}
\input{chapters/related_work}
\input{chapters/overview}
这三章时，第二章会接着第一章结尾开始，而不用另起一页？找了很多方法都不能很好的解决这个问题。
有个方法是在正文前输入
\renewcommand{\cleardoublepage}{}
\renewcommand{\clearpage}{}
但是这又使得PDF输出不全。

**Comment by iori2333** [comment]
你有没有用 `\chapter{标题}` 来生成章标题呢？

------

我倒是能搜到：<https://tex.stackexchange.com/questions/24066/start-new-chapter-on-same-page>

在 `\input` 前后加上这样的命令就可以。

```
\begingroup
\let\cleardoublepage\relax % 关闭另起一页的命令
\input{chapters/introduction}
\input{chapters/related_work}
\input{chapters/overview}
\endgroup
```

**Reply by EF-Lin233** [reply]
我按照注释里写的，新建一个 chapters 文件夹,把每一章的内容单独放入一个 tex 文件,然后在main.tex里用 \input{} 导入

**Reply by iori2333** [reply]
回答已更新。

**Reply by EF-Lin233** [reply]
这个方法我试过，确实可以将每章连在一起，但是输出pdf文字会错位（可能是用了模板的缘故？）

**Reply by iori2333** [reply]
请截个图描述你所说的”错位“。我测试的例子是没问题的。而且，新章不换页不符合这个模板遵循的排版标准。

```
\documentclass{njuthesis}
\usepackage{zhlipsum}
\ctexset{
    chapter/beforeskip=10pt,
    chapter/afterskip=0pt
}
\begin{document}
\mainmatter
\begingroup
\let\cleardoublepage\relax
\chapter{1}
\section{1}
\zhlipsum[1-10][name=zhufu]
\chapter{1}
\section{1}
\zhlipsum[11-20][name=zhufu]
\endgroup
\end{document}
```
